### DISCUSSION TITLE
Contact Form

### DISCUSSION THREAD

**Discussion by aloscher** [vanjarosoftware/Vanjaro.Platform#1594] [Ideas]
I love this new platform. The one thing I think is a critical missing is a basic contact form. In another discussion I read that Live Forms will be available from a Vanjaro web store which is excellent, but if I have a site the only requires a contact us form, purchasing a module seems like overkill. Every site I have ever built has a contact form and many have no other forms.

Do you think this could be considered in the future?

**Comment by Tychodewaard** [comment]
Besides Live Forms, there will be a form module available in the Vanjaro Store but pricing is not available atm. As all modules can be installed, you could consider free alternatives like OpenForm (https://dnncommunity.org/Forge)

**Reply by aloscher** [reply]
I tried the Form and List module and it doesn't work well.  I prefer DNN Dev's Xmod Pro to make forms but my hope was to build only one portal per install with Vanjaro and the cost of using Xmod Pro or Live Forms for a single form is a bit of overkill and not so cost effective. Hope to find a workable solution as a contact form is quite standard. What do you use for this?

**Reply by Tychodewaard** [reply]
I use ActionForm or Live Forms because I have Plantanapp and a Mandeeps suite. So, there are no extra costs for me.

**Comment by meetmandeep** [comment]
Vanjaro Platform does not ship with any apps (modules). From a strategical perspective, we will not bundle any apps with the platform. It's easy to start with a simple form but then feature requests keeps coming in and the focus is drifted from the core platform and its capabilities to specific use cases. 

Vanjaro App Store will be released shortly to fill in this gap and create an ecosystem. Both commercial and free apps will be available on App Store; anyone in the community can contribute. As @Tychodewaard mentioned, all existing DNN Modules are also compatible with Vanjaro and you could use any one of those as well.

**Reply by aloscher** [reply]
I tried Forms and lists and another module today and they were not behaving well. I would love to use live Forms but my intention is to build my smaller sites with Vanjaro and only one portal.  I love your products (Mandeeps Products) and will use them in DNN without Vanjaro. I am excited about Vanjaro as a stand-a-lone but cannot develop until I have a simple solution for a basic contact form. For now I will keep playing with the software. Thanks for this development. If there can be some cost effective simple apps as well as more complex, that would work for what I am trying to accomplish. 

Thanks for your message
